web_src/js/components/RepoActionView.vue

```diff
@@ -165,9 +165,12 @@ const sfc = {
   mounted() {
     // load job data and then auto-reload periodically
     this.loadJob();
-    this.intervalID = setInterval(this.loadJob, 1000);
     // load artifactss list once
     this.loadArtifacts();
+    this.intervalID = setInterval(() => {
```

Comment:
I do not think this is right. `intervalID` is controlled by `loadJob`, there is clearInterval

Reply:
I think this is also missing a `clearInterval` when the component unmounts. We should do it during [unmounted](https://vuejs.org/api/options-lifecycle.html#unmounted) lifecycle.